service/storage_proxy.hh

```diff
@@ -158,6 +158,8 @@ public:
         NONE,
         TIMEOUT,
         FAILURE,
+        DISCONNECTED,
+        OVERLOADED,
```

Comment:
I really think we need to explain the meaning of all these errors somewhere. What does "DISCONNECTED" mean, disconnected from what?